src/applications/vaos/components/NewTabAnchor.jsx

```diff
@@ -10,8 +10,9 @@ import PropTypes from 'prop-types';
  * message to announce that the link opens in a new window.
  *
  * @export
- * @param {String} href The URL that the hyperlink points to.
- * @param {String} children Text describing the link destination.
+ * @param {Object} params
```

Comment:
I accidentally made this change in here, but it seems simple enough that it's not worth removing